examples/bookstore/bookstore.proto

```diff
@@ -36,34 +36,41 @@ message Error {
   string message = 2;
 }
 
+//CreateShelfParameters holds parameters to CreateShelf
 message CreateShelfRequest {
   Shelf shelf = 1;
 }
 
+//GetShelfParameters holds parameters to GetShelf
```

Comment:
Should we use the same comments as the bookstore protos in Google examples? https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/endpoints/bookstore-grpc/bookstore.proto